logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/register/VerifyActivationCodeUseCase.kt

```diff
@@ -0,0 +1,37 @@
+package com.wire.kalium.logic.feature.register
+
+import com.wire.kalium.logic.NetworkFailure
+import com.wire.kalium.logic.configuration.ServerConfig
+import com.wire.kalium.logic.data.register.RegisterAccountRepository
+import com.wire.kalium.network.exceptions.KaliumException
+import com.wire.kalium.network.exceptions.isInvalidCode
+
+class VerifyActivationCodeUseCase(
+    private val registerAccountRepository: RegisterAccountRepository
+) {
+    suspend operator fun invoke(email: String, code: String, serverConfig: ServerConfig): VerifyActivationCodeResult {
+
+        return registerAccountRepository.verifyActivationCode(email, code, serverConfig.apiBaseUrl)
```

Comment:
The same body definition applied here. Just in case if you agree with me then we need to change this.